integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/ephemeral/DistroXUpgradeTests.java

```diff
@@ -56,6 +57,7 @@ protected void setupTest(TestContext testContext) {
         createEnvironmentWithFreeIpa(testContext);
     }
 
+    @Ignore("Yarn config verification is not working, because CM responds with a server error")
```

Comment:
@aszegedi told me that ignore doesn't help es skipped test would causes failure for the job too